docs/user_guide/mitigation.rst

```diff
@@ -68,6 +68,263 @@ Postprocessing
+    ...     constraints="demographic_parity",
+    ...     objective="accuracy_score")
+    >>> threshold_optimizer.fit(X, y, sensitive_features=sex)
+    ThresholdOptimizer(estimator=LogisticRegression())
+    >>> print(json.dumps(threshold_optimizer.thresholds(), default=str, indent=4))
+    {
+        "Female": {
+            "p0": 0.9527202201546324,
+            "operation0": "[>0.5]",
+            "p1": 0.047279779845367575,
+            "operation1": "[>-inf]"
+        },
+        "Male": {
+            "p0": 0.9998874603553525,
+            "operation0": "[>0.5]",
+            "p1": 0.00011253964464752464,
+            "operation1": "[>-inf]"
+        }
+    }
+    >>> plot_threshold_optimizer(threshold_optimizer)
+
+Notice that the thresholding rules do not include :math:`p_{\text{ignore}}` which is
+only relevant for :code:`equalized_odds`. The results can be interpreted as
+follows:
+
+- "Male":
+
+  - if the score is less or equal to :math:`0.5` predict 1 with probability
+    :math:`0.00011`
+  - if the score is above :math:`0.5` predict 1
+
+- "Female:
+
+  - if the score is less or equal to :math:`0.5` predict 1 with probability
+    :math:`0.04728`
+  - if the score is above :math:`0.5` predict 1
+
+The thresholds are actually the same for both groups here, which
+does not always have to be true. As a consequence, the only difference is
+the probability to predict label 1 in the case where the score is below
+:math:`0.5`.
+
+Note that the chart is different from the chart for :code:`equalized_odds`.
+It shows :code:`selection_rate` (which is the basis for our fairness
+criterion :code:`demographic_parity`) on the x-axis, and
+:code:`accuracy_score` (which is the basis for the implicit performance
+objective) on the y-axis.
+
+.. note::
+
+    :class:`ThresholdOptimizer` expects an estimator that provides it with
+    scores. While the output of :class:`ThresholdOptimizer` is binary, the
+    input need not be. In fact, real valued input, e.g. from a regressor,
+    provides it with many more options to create thresholds. For :math:`n`
+    input data points with :math:`m \leq n` different score values it has
+    :math:`m+1` different thresholds. At each threshold one can create one of
+    two thresholding rules, i.e. functions that indicate which data points
+    get label :code:`1` based on their score.
+
+The following combinations of fairness criteria and objectives are available:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   *  - fairness criteria
+      - objectives
+   *  - :code:`equalized_odds`
+      - :code:`accuracy_score`, :code:`balanced_accuracy_score`
+   *  - :code:`demographic_parity`, :code:`false_positive_rate_parity`, :code:`false_negative_rate_parity`, :code:`true_positive_rate_parity`, :code:`true_negative_rate_parity`
+      - :code:`selection_rate`, :code:`true_positive_rate`, :code:`true_negative_rate`, :code:`accuracy_score`, :code:`balanced_accuracy_score`
```

Comment:
could these lines be broken?

Reply:
How do you mean? It renders like this (as intended):
![image](https://user-images.githubusercontent.com/10245648/103592344-b2ba7080-4ea7-11eb-9c23-2d4a19f99e1e.png)

Reply:
i'm assuming @adrinjalali is asking for shorter lines in `.rst`

Reply:
Ah, didn't realize that I can move it over several lines! Done!